### PR TITLE
Adding Orbit-Bundle for Eclipse Paho MQTT Client Library

### PR DESCRIPTION
**Pull request by Shabirmean**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -61,6 +61,7 @@
         <module>lucene/4.10.3-wso2v1</module>
         <module>rabbit-mq/3.4.4.wso2v1</module>
         <module>scannotation/1.0.3.wso2v1</module>
+        <module>eclipse-mqtt-paho/1.0.2.wso2v1</module>
```
Comment (Nipuni) [inline pom.xml]: Why this module is added to parent pom. This is not needed as we release orbit bundles individually.

File: eclipse-mqtt-paho/1.0.2.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+	<url>http://wso2.org</url>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.eclipse.paho</groupId>
+			<artifactId>org.eclipse.paho.client.mqttv3</artifactId>
+			<version>${eclipse.paho.version}</version>
+		</dependency>
+	</dependencies>
+
```
Comment (Nipuni) [inline eclipse-mqtt-paho/1.0.2.wso2v1/pom.xml]: Mark the dependencies optional

File: pom.xml (hunk 1 above)
Comment (Shabirmean) [inline pom.xml]: Sorry. I have removed it from the parent pom.

File: eclipse-mqtt-paho/1.0.2.wso2v1/pom.xml (hunk 2 above)
Comment (Shabirmean) [inline eclipse-mqtt-paho/1.0.2.wso2v1/pom.xml]: I have updated the dependency to optional

**Comment by Shabirmean** [comment]
Closed this PR since unable to update the indentation from tab to spaces. Please find new PR at: https://github.com/wso2/orbit/pull/167

**Pull request closed without merging**